examples/src/shader_f16/README.md

```diff
@@ -0,0 +1,9 @@
+# shader-f16 
+
+Demonstrate the ability to perform compute in F16 using wgpu.
+
+## To Run
+
+```
+RUST_LOG=hello_compute cargo run --bin wgpu-examples shader_f16 
```

Comment:
question: Why is `RUST_LOG` set to `hello_compute`, which is the name of a different example? Is this a typo?

Reply:
Copy paste error!

Reply:
Fixed to use `shader_f16` instead.